modules/about-developer-perspective.adoc

```diff
@@ -15,6 +15,8 @@ The *Developer* perspective offers several built-in ways to deploy applications,
 
 Cluster administrators can also open an embedded command-line terminal instance in the web console in {product-title} 4.7 and later.
 
+include::snippets/unified-perspective-web-console.adoc[]
```

Comment:
Similar comment here about being careful to not place admonitions too close to each other.